react/features/toolbox/actions.native.js

```diff
@@ -4,6 +4,7 @@ import type { Dispatch } from 'redux-thunk';
 
 import {
     CLEAR_TOOLBOX_TIMEOUT,
+    SET_DEFAULT_TOOLBAR_BUTTONS,
```

Comment:
All the actions defined in this file seem to me related to web and not native. Why are they in actions.native.js?